app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,69 @@
+require 'pry'
+class TweetController < ApplicationController
+
+  get "/tweets" do
+    if logged_in?
+      @user = current_user
+      erb :"/posts/index"
+    else
+      redirect to "/login"
+    end
+  end
+
+  get "/tweets/new" do
+    if logged_in?
+      erb :"/posts/new"
+    else
+      redirect to "/login"
+    end
+  end
+
+  post "/tweet" do
+    if params[:content] != ""
```

Comment:
I think you could use [ActiveRecord validations](http://guides.rubyonrails.org/active_record_validations.html) (e.g. `validates_presence_of :username`) to perform these types of checks more easily.